builds/azure-pipelines/build-release.yml

```diff
@@ -4,7 +4,7 @@ parameters: # parameters are shown up in ADO UI in a build queue time
 - name: 'ReleaseType'
   displayName: 'Type of release'
   type: string
-  default: full
+  default: preview
```

Comment:
Is the plan to have these be preview packages until we do the full release? If so please make a task to track this

Reply:
Make a task to release the full version instead of the preview once everything is ready?

Reply:
Yes (if we don't have one already)

You can probably consolidate all these getting-ready-for-GA tasks into a single issue to make them easier to track, since most of them are pretty small and can do be done at the same time.